src/modules/awake/Awake/Properties/Resources.resx

```diff
@@ -123,32 +123,38 @@
   <data name="AWAKE_EXIT" xml:space="preserve">
     <value>Exit</value>
   </data>
-  <data name="AWAKE_HOUR" xml:space="preserve">
-    <value>hour</value>
+  <data name="AWAKE_1_HOUR" xml:space="preserve">
+    <value>1 hour</value>
   </data>
   <data name="AWAKE_HOURS" xml:space="preserve">
-    <value>hours</value>
+    <value>{0} hours</value>
+    <comment>0: A number grater than 1</comment>
```

Comment:
Same as below. You can also add in the comments that {0} will be replaced at runtime and shouldn't be removed.